File: samples/standalone/sdk-extension/src/test/java/com/probable/guacamole/MockTenant.java
/* SPDX-License-Identifier: Apache-2.0
   Copyright 2023 Atlan Pte. Ltd. */
package com.probable.guacamole;

import com.atlan.mock.MockAtlanTenant;
import org.testng.annotations.AfterSuite;
import org.testng.annotations.BeforeSuite;

public class MockTenant extends MockAtlanTenant {
    @BeforeSuite
    void start() throws InterruptedException {
        initializeClient();
    }

    @AfterSuite
    void stop() {
        closeClient();
    }
}

File: package-toolkit/runtime/src/test/kotlin/com/atlan/mock/MockTenant.kt
/* SPDX-License-Identifier: Apache-2.0
   Copyright 2023 Atlan Pte. Ltd. */
package com.atlan.mock

import org.testng.annotations.AfterSuite
import org.testng.annotations.BeforeSuite

class MockTenant : MockAtlanTenant() {
    @BeforeSuite
    fun start() {
        createClient()
    }

    @AfterSuite
    fun stop() {
        closeClient()
    }
}
